plugins/node/opentelemetry-instrumentation-restify/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@opentelemetry/instrumentation-restify",
+  "version": "0.14.0",
+  "description": "OpenTelemetry restify automatic instrumentation package",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.ts'",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "tdd": "yarn test -- --watch-extensions ts --watch",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "version:update": "node ../../../scripts/version-update.js",
+    "compile": "npm run version:update && tsc -p .",
+    "prepare": "npm run compile",
+    "watch": "tsc -w"
+  },
+  "keywords": [
+    "opentelemetry",
+    "restify",
+    "nodejs",
+    "tracing",
+    "instrumentation"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.5.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@opentelemetry/context-async-hooks": "0.18.0",
+    "@opentelemetry/node": "0.18.0",
+    "@opentelemetry/tracing": "0.18.0",
+    "@types/lodash.once": "^4.1.6",
+    "@types/mocha": "7.0.2",
+    "@types/node": "14.0.27",
+    "@types/restify": "^4.3.7",
+    "codecov": "3.7.2",
+    "gts": "3.1.0",
+    "mocha": "7.2.0",
+    "nyc": "15.1.0",
+    "restify": "^4.3.4",
+    "rimraf": "3.0.2",
+    "ts-mocha": "8.0.0",
+    "ts-node": "9.0.0",
+    "tslint-consistent-codestyle": "1.16.0",
+    "tslint-microsoft-contrib": "6.2.0",
+    "typescript": "4.1.3"
+  },
+  "dependencies": {
+    "@opentelemetry/api": "^0.18.0",
+    "@opentelemetry/core": "^0.18.0",
+    "@opentelemetry/instrumentation": "^0.18.0",
+    "@opentelemetry/semantic-conventions": "^0.18.0",
+    "lodash.once": "^4.1.1"
```

Comment:
I would like to remove any 3rd party dependency

Reply:
I get where you are coming - I'd like to avoid adding any deps to instrumentations whenever possible. I would not write a `lodash` utility function myself either, because I would most certainly do a worse job at this than lodash developers. It's a very well-trusted library and lightweight one-function-package.

More importantly: do we actually need this? I'm using it to avoid calling `span.end()` multiple times, but feel like `span.end()` should be noop anyways after the first time it's called.

Reply:
avoiding any dependency in instrumenting has much more sense if you think that any other library can be instrumented as well. We also in general avoid any 3rd party dependency unless this is really necessary and provides a core functionality. Here you just want to prevent the span.end being called twice which can be done without it.

Reply:
Clear! I'll just remove it then and live with having the warning messages.